x/evm/types/config.go

```diff
@@ -2,16 +2,18 @@ package types
 
 // EVMConfig contains all EVM module configuration values
 type EVMConfig struct {
-	Name       string `mapstructure:"name"`
-	RPCAddr    string `mapstructure:"rpc_addr"`
-	WithBridge bool   `mapstructure:"start-with-bridge"`
+	Name               string `mapstructure:"name"`
+	RPCAddr            string `mapstructure:"rpc_addr"`
+	WithBridge         bool   `mapstructure:"start-with-bridge"`
+	EnableRPCDetection bool   `mapstructure:"enable-rpc-detection"`
```

Comment:
Just wanna confirm before you merge, does this mean that `EnableRPCDetection` is optional in the `config.toml` file and if one doesn't have it there it would default to `true`?